lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +8,21 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ * @property {any} body The cast JSON -> object body element
+ */
+export interface ISailplaneAPIGatewayProxyEvent extends APIGatewayProxyEvent {
+    /**
+     * Cast JSON -> object body element
+     * @property body
+     * @type {any}
+     */
+    body: any;
+}
```

Comment:
Since LambdaUtils also de-nulls `queryStringParameters` and `pathParameters`, can do those too:
```
    pathParameters: { [name: string]: string };
    queryStringParameters: { [name: string]: string };
```